nix/web-security-tracker.nix

```diff
@@ -235,6 +235,8 @@ in
           wantedBy = [ "multi-user.target" ];
 
           script = ''
+            # Before starting, crash all the in-progress evaluations.
```

Comment:
```suggestion
            # Before starting, crash all the in-progress evaluations.
            # This will prevent them from being stalled forever, since workers would not pick up evaluations marked as in-progress.
```